deployment/README.md

```diff
@@ -0,0 +1,74 @@
+# Amazon Web Services Deployment
+
+Amazon Web Services deployment is driven by [Terraform](https://terraform.io/), [Ansible](https://www.ansible.com/), and the [AWS Command Line Interface (CLI)](http://aws.amazon.com/cli/).
+
+## Table of Contents
+
+* [Deployment](#deployment)
+    * [Pre-Deployment Configuration](#pre-deployment-configuration)
+    * [SSH Keys](#ssh-keys)
+    * [DB_SETTINGS_BUCKET](#db_settings_bucket)
+    * [User Data](#user-data)
+    * [`scripts/infra`](#scriptsinfra)
+    * [Loading Shapefile data](#loading-shapefile-data)
+
+## Deployment
+
+### Pre Deployment Configuration
+
+#### SSH Keys
+
+You'll also need to generate an SSH Keypair using the AWS EC2 console. Download the private key, and store it at `~/.ssh/district-builder.pem`.
+
+#### DB_SETTINGS_BUCKET
```

Comment:
We've usually tried to avoid this abbreviation for this project in a code context just because it's so easy to get confused with "database".